lib/web_ui/lib/src/engine/semantics/tappable.dart

```diff
@@ -16,6 +16,9 @@ class SemanticButton extends SemanticRole {
     setAriaRole('button');
   }
 
+  @override
+  bool get acceptsPointerEvents => true;
```

Comment:
Should the `Tappable` behavior below return `true` instead? This way it can be inherited by all nodes that are tappable, even those that are not buttons.

Reply:
🤦‍♂️ that's what I meant to do. Thanks for catching it!